megamek/src/megamek/client/ui/swing/lobby/PlayerSettingsDialog.java

```diff
@@ -179,6 +188,7 @@ private void setupDialog() {
             mainPanel.add(mineSection());
         }
         mainPanel.add(skillsSection());
+        mainPanel.add(emailSection());
```

Comment:
Can we make adding this element conditional on the player not being a bot?

Reply:
Fixed - this is now only shown for non-bots.

Reply:
Oh, I missed the part about the case where a player disconnects and is replaced by a bot (didn't even realise that was possible!) - will look into that tonight.

Reply:
latest pish ensures that bots don't get emailed either